challengeutils/__main__.py

```diff
@@ -137,6 +138,9 @@ def build_parser():
         default=synapseclient.client.CONFIG_FILE,
         help="credentials file")
 
+    parser.add_argument('-v', '--version', action='version',
+                        version='Synapse Client {}'.format(__version__))
```

Comment:
Aren't challengeutils and synapse client separate things? If you're adding versioning for the former in this PR, why does this argument say Synapse Client?

Reply:
Copied this argparse from `synapseclient`.  Thanks for the catch!